sdks/java/container/build.gradle

```diff
@@ -60,6 +61,8 @@ task copyDockerfileDependencies(type: Copy) {
   rename 'beam-sdks-java-harness-.*.jar', 'beam-sdks-java-harness.jar'
   rename 'beam-sdks-java-io-kafka.*.jar', 'beam-sdks-java-io-kafka.jar'
   rename 'kafka-clients.*.jar', 'kafka-clients.jar'
+  rename 'beam-sdks-java-io-snowflake.*.jar', 'beam-sdks-java-io-snowflake.jar'
```

Comment:
I'm not sure if it wouldn't be ok to add *SNAPSHOT.jar instead of .jar because when built as not shadow then the name of the jar is *-unshaded.jar

Reply:
Done